script.js

```diff
@@ -0,0 +1,87 @@
+const submitForm = document.querySelector('.input-box'); // 할 것 제출
+const button = document.querySelector('button'); // 입력버튼
+const textInput = document.querySelector('input'); // 추가 할 내용
+const todolist = document.querySelector('#todo-list'); // 버튼을 눌렀을 때 추가되는 곳
+const todo = document.querySelector('.todo');
+const done = document.querySelector('.done');
+const doneList = document.querySelector('#done-list');
+var count_todolist = 0;
+var count_donelist = 0;
+
+var localStorage = [];
+
+// 폼 제출 비동기 함수
+submitForm.addEventListener('submit', (e) => {
+    e.preventDefault();
+    console.log('hi');
+    if (textInput.value) {
+        // textInput.value에 내용이 담겼으면
+        const div = document.createElement('div');
+        const span = document.createElement('span');
+        span.textContent = textInput.value;
+        // div 안 span에 input 내용을 넣어줌
+        span.classList.add('content');
+        span.style.display = 'inline-block';
+        span.style.width = '250px';
+        div.append(span);
+
+        // double click로 todo -> done, done -> todo 구현하는 event를 span에 적용
+        span.addEventListener('dblclick', (e) => {
+            console.log(e.target.parentNode.id);
+            const parent = e.target.parentNode;
+            if (parent.parentNode.id === 'todo-list') {
+                doneList.append(e.target.parentNode);
+                span.style.textDecoration = 'line-through';
+                count_donelist += 1;
+                count_todolist -= 1;
+            } else {
+                todolist.append(e.target.parentNode);
+                span.style.textDecoration = 'none';
+                count_donelist -= 1;
+                count_todolist += 1;
+            }
+            todo.textContent = 'To do list (' + count_todolist + ')';
+            done.textContent = 'Done list (' + count_donelist + ')';
+        });
+
+        // console.log('div content', div.textContent);
+        // console.log(textInput.value);
+
+        // 삭제 버튼 만들기
+        const button_span = document.createElement('span');
+        const button = document.createElement('button');
```

Comment:
변수 네이밍의 일관성을 지켜주세요~~ `lowerCamelCase`로 해주시고, 좀 더 의미가 잘 드러나게 지어주셨으면 좋겠습니다.

```suggestion
        const buttonSpan = document.createElement('span');
        const deleteButton = document.createElement('button');
```

Reply:
다 수정 한다는걸 깜빡했습니다!! ㅠ.ㅠ 고치겠습니다